### PR TITLE
Add a link to COOP/COEP guide, embed two videos

### PR DESCRIPTION
**Pull request by agektmr**
- Adds a link to [A guide to enable cross-origin isolation](https://web.dev/cross-origin-isolation-guide/).
- Embeds two video about COOP/COEP from web.dev live and CDS.

### CODE REVIEW

**Comment by agektmr** [comment]
Looks like I can assign but cc'ing @lutzvahl

**Comment by lutzvahl** [comment]
> Looks like I can assign but cc'ing @lutzvahl

Thanks, content looks good. LGTM

**Comment by agektmr** [comment]
@mihajlija Once you LGTM we can merge this.

File: src/site/content/en/blog/coop-coep/index.md [hunk 1]
```diff
@@ -104,6 +106,11 @@ clearer, let's define them first:
 
 ## Deploy COOP and COEP to make your website cross-origin isolated
 
+{% Aside 'gotchas' %}
```
Comment (mihajlija) [inline src/site/content/en/blog/coop-coep/index.md:109]:
```suggestion
{% Aside %}
```

Comment (mihajlija) [reply src/site/content/en/blog/coop-coep/index.md:109]: `gotchas` are for "a common problem that the reader wouldn't know without specialized knowledge of the topic", this is just additional information so it should be a regular aside. https://web.dev/handbook/web-dev-components/#asides

**Review by mihajlija — APPROVED** [review]
LGTM w/ comments, please take a look and merge once resolved.

**Pull request merged**